### ISSUE TITLE
When a tracks seconds from midnight include pass through zero sorting  on time will fail

### ISSUE THREAD

**Issue by robgf** [robgf/Clean_analyze_survey_tracks#10]
When we impute time we will want t o do a shift so that small seconds from midnight that happen after large seconds from midnight are sorted correctly.

Example:
EffortID == 377
max(SeconcdsFromMidnight) = 1315931.38 and 
min(SecondsFromMidnight) = 0.00

Solution for now can be to add 5 to 9 hours to time and divide by 24*3600.

**Comment by robgf** [comment]
Opps! 1315931.38 isn't even a reasonable time from midnight. As it is over 365 days.  This may be an artifact from calculating time to midnight on a particular date just in the wrong year?

**Comment by thoufer** [comment]
max(SeconcdsFromMidnight)  should be 86399.. as 86400 would be midnight of
the next day.


---------------------------------------
US Fish and Wildlife Service
Division of Migratory Bird Management
11510 American Holly Dr.
Laurel MD 20708

301-497-5869 (phone)
301-497-5871 (fax)
https://www.fws.gov/birds

**Comment by embuco** [comment]
Yes, we never have that problem unless the time is in GMT (but then it is not time since midnight ....)
